app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1227,7 +1234,11 @@ class BrowserTabFragment :
         webView = null
     }
 
-    private fun requestFileDownload(url: String, contentDisposition: String, mimeType: String, requestUserConfirmation: Boolean) {
+    private fun convertBlobToDataUri(blob: Command.ConvertBlobToDataUri) {
+        blobConverterInjector.convertBlobIntoDataUriAndDownload(webView as WebView, blob.url, blob.mimeType)
```

Comment:
`webView as WebView` reads strangely here. I assume this is because we have a nullable `webView` and are passing it into a function that wants non-nullable type.

Could we do the null check instead (or `webView?.let {}`) which is probably clearer?

Reply:
Done!